test/Microsoft.Health.SqlServer.Web/Properties/launchSettings.json

```diff
@@ -3,11 +3,7 @@
     "Microsoft.Health.SqlServer.Web": {
       "commandName": "Project",
       "environmentVariables": {
-        "ASPNETCORE_ENVIRONMENT": "development",
-        "SqlServer:Initialize": "true",
```

Comment:
I am wondering if we could make Initialize and allowDatabaseCreation as false and not remove these. And when we plan to create a database to connect to, we can flip them on. I use these settings frequently for my local testing so just gave it a thought but it could be just me :)

Reply:
Sure!

Reply:
To make sure I understand, if we would like to perform an auto schema upgrade for an existing database, we will need to flip `SqlServer:Initialize` to be true, but if we would like to perform a manual schema upgrade, this won't be needed - correct?

If so, it might be worthwhile to make an investigate story and dig into these config options a bit more. It would be good to ensure their functionality is clear and consistent.